linux/signal_tools.bsh

```diff
@@ -77,7 +77,7 @@ function trap_chain()
 
     # Since there was no signal already assigned, the sanitize signal name is
     # not in last_info, so we'll have to make it
-    last_info="$(trap '' "${2}"; trap -p "${2}" | { read x ; echo "${x##* }"; })"
+    last_info="$((trap '' "${2}"; trap -p "${2}") | { read x ; echo "${x##* }"; })"
```

Comment:
Add comment that () fixed #242, or else I might delete it in the future by accident.

or... How hard would it be to add a unit test to test that this works?